packages/caip/src/adapters/mtpelerin/index.ts

```diff
@@ -69,13 +79,6 @@ const MtPelerinSymbolToAssetIds: Record<string, AssetId[]> = {
  * The following is the list of assets that Mt Pelerin supports
  * but they're **not supported** (yet?) in ShapeShift
  * {
-    '[xdai_mainnet]/0x0000000000000000000000000000000000000000': 'XDAI',
-    '[xdai_mainnet]/0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83': 'USDC',
-    '[xdai_mainnet]/0x4ECaBa5870353805a9F068101A40E0f32ed605C6': 'USDT',
-    '[xdai_mainnet]/0x8e5bBbb09Ed1ebdE8674Cda39A0c169401db4252': 'WBTC',
-    '[xdai_mainnet]/0x6A023CCd1ff6F2045C3309768eAd9E68F978f6e1': 'WETH',
```

Comment:
What about this one?

<img width="548" alt="image" src="https://github.com/shapeshift/web/assets/17035424/3233c8ea-08dc-4653-ac59-efbd5540fc8a">

Reply:
https://github.com/shapeshift/web/pull/4585/commits/de386f694771decf4d99e0fb970eee05cbed57ed